docs/pages/released-label.md

```diff
@@ -0,0 +1,60 @@
+# Released Label Plugin
+
+This plugin
+
+- comments on the merged PR with the new version
+- comments on closed issues with the new version
+- adds a `released` label to the pull request
+
+::: message is-warning
+Make sure that you create the `released` label on you project
```

Comment:
Wait, does it still expect you to manually add the label?

Reply:
actually after testing GitHub seems to have created the label. I opened #254 to try to tackle this for `create-labels`